### PR TITLE
tribute page review

### PR DESCRIPTION
**Pull request by Richez003**
made some changes to my tribute page

### CODE REVIEW

**Review by vic778 — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @Richez003 ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure your implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._

- [x] kindly you have submitted a project that does not comply with the requirements of the project, you should review your project and leave some commentary as it is well indicated on [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,46 +1,64 @@
...
-<body> 
-
-  <section class="intro">
-    <div>
-  <h1>NELSON MANDELA</h1>
-    <img src="./image/nelson.webp" alt="">
-
-    <p>“Do not judge me by my successes, judge me by how many times I fell down and got back up again.”</p>
+<html>
+     <!-- < link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,300&display=swap" rel="stylesheet"> -->
```
Comment (Richez003) [inline index.html:3]: I think your links shouldn't be in a comment for easy access to the link on the page

File: index.html [hunk 2]
```diff
@@ -1,46 +1,64 @@
...
+      <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+      <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+      <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+      <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+      <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
```
Comment (Richez003) [inline index.html:46]: **Why don't you use a paragraph tag instead of a list tag for better structuring of your write up**

File: index.html [hunk 3]
```diff
@@ -1,46 +1,64 @@
...
+  <head>
+      
+    <style>
+    <title id="NELSON MANDELA"</title> 
+    </style>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <main id="main">
+  <body>
+    <!--TRIBUTE TITLE-->
```
Comment (Richez003) [inline index.html:13]: **I think this comment should be removed as not to cluster your work**

File: index.html [hunk 4]
```diff
@@ -1,46 +1,64 @@
...
+        <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+        <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+        <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+       <li>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Distinctio voluptates incidunt ut eveniet quas doloremque voluptatum veritatis velit numquam iste officia vero fugit, esse cum nesciunt at odit, repellendus aliquam?</li>
+       </ul>
+    <!--External link for more Mandela info-->
```
Comment (Richez003) [inline index.html:55]: **I think this comment should be removed as to give space for important codes to avoid clustering**

File: index.html [hunk 5]
```diff
@@ -1,46 +1,64 @@
...
-
- <p>Do you want to know more about <a href="https://pl.wikipedia.org/wiki/Nelson_Mandela">Nelson Mandela</a>
-</p>
-</section>
-</body>
-</html>
\ No newline at end of file
+  </body>
+  </main>
+</html>
```
Comment (Richez003) [inline index.html:62]: **I think it will be better you use a semantic tags for better structuring of you Html**

File: index.html [hunk 6]
```diff
@@ -1,46 +1,64 @@
...
+index.html
+Displaying index.html.
```
Comment (Richez003) [inline index.html:64]: **I think this is an error and should be removed**

File: index.html [hunk 7]
```diff
@@ -1,46 +1,64 @@
...
+    </h1>
+     <!-- tribute person image-->
+    <div class="image">
+    <div id="img-div">
+    <img id="image" src="https://cdn.britannica.com/93/173193-131-3EE3B458/Nelson-Mandela-South-African.jpg" alt="photo of Nelson Mandela">
+    <!--figure caption-->
+      <p id="img-caption">“Do not judge me by my successes, judge me by how many times I fell down and got back up again.”</p>
+    </div>
+    </div>
+  <!--Mandela tribute info-->
```
Comment (Richez003) [inline index.html:24]: **I  think line 24 should be remove too as explained earlier**

File: index.html [hunk 8]
```diff
@@ -1,46 +1,64 @@
...
+  <main id="main">
+  <body>
+    <!--TRIBUTE TITLE-->
+    <h1 id="title">NELSON MANDELA
+    </h1>
+     <!-- tribute person image-->
+    <div class="image">
+    <div id="img-div">
+    <img id="image" src="https://cdn.britannica.com/93/173193-131-3EE3B458/Nelson-Mandela-South-African.jpg" alt="photo of Nelson Mandela">
+    <!--figure caption-->
```
Comment (Richez003) [inline index.html:20]: **I also think this should be removed as earlier explained**

**Review by Richez003 — COMMENTED** [review]
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f52326d324e7a567851337062472f67697068792e676966

Required Changes ♻️

Use the correct gitwork flow
Run linters to check for errors 
Check the comments under the review.

**Optional suggestions**
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @Richez003 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Required Changes ♻️
- [x] Kindly phrase the comments politely, as suggestions instead of orders.
- For example:
   - This is a polite suggestion: "I think it would be better to use semantic tags"
   - This reads like an order: "Use semantic tags"

- [ ] Kindly try to explain why you request the changes.
- For example:
   - This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
   - This request lacks a "why": Why don't you use an article tag here instead of a div?

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Richez003 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**